.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: required
```

Comment:
Are we using travis?

Reply:
I think that's an old artefact. Will fix it in the next commit.